app/pathogen/generic-pathogen-visualizations-page.tsx

```diff
@@ -8,6 +8,12 @@ interface GetDisplayNameInput<TEstimate extends Record<string, unknown>> {
   data: TEstimate[]
 }
 
+interface RenderVisualizationInput<TEstimate extends Record<string, unknown>> {
+  data: TEstimate[];
+  highlightedDataPoint: TEstimate | undefined;
+  hideArbovirusDropdown: boolean | undefined;
```

Comment:
Ideally these two wouldn't live in the generic component. This bothers me a bit but my attempted fix was a pretty gross generic thing that reads the input parameter of the render function and it was just way too much. Maybe later but for right now we'll have to tolerate this I guess :'(